.yarnrc.yml

```diff
@@ -0,0 +1,4 @@
+yarnPath: .yarn/releases/yarn-sources-pr-2587.cjs
+pnpIgnorePatterns:
+  - docs/**
+  - packages/**
```

Comment:
This would ideally be simplified to `*/**` (combine `docs` and `packages` to `*` to avoid the need to list subfolders explicitly).

There seems to be some internal issue why `*/**` doesn't work at this point – it's something about the trailing slash removal in the generated regex which is

```
(^(?:\.yarn\/sdks(?:\/(?!\.{1,2}(?:\/|$))(?:(?:(?!(?:^|\/)\.{1,2}(?:\/|$)).)*?)|$))$)|(^(?:(?!\.{1,2}(?:\/|$))(?=.)[^/]*?\/(?!\.{1,2}(?:\/|$))(?:(?:(?!(?:^|\/)\.{1,2}(?:\/|$)).)*?))$)
```

See [this Discord message](https://discord.com/channels/226791405589233664/654372321225605128/825822800580182026) by @merceyz and a few messages above and below it.